testsuite/cli/libra-wallet/src/key_factory.rs

```diff
@@ -104,6 +104,11 @@ impl ExtendedPrivKey {
         libra_types::account_address::from_public_key(&self.get_public())
     }
 
+    /// Get private key
+    pub fn get_private_key(&self) -> Ed25519PrivateKey {
+        self.private_key.clone()
```

Comment:
looks like you'll need to return a reference or perhaps you want to relabel this to something more insidious such as get_private_key_value and convert it into bytes

Reply:
I just enabled `features = ["fuzzing"]` for libra-crypto.